### PR TITLE
added  4 problems of Codewars

### PR DESCRIPTION
**Pull request by soham4abc**
Four new problems added of Codewars Platform. More on the way!

### CODE REVIEW

File: Codewars/Leap_years.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        else
+        {
+            return false;
+        }
+    }
+    else
+    {
+        if (y % 4 == 0)
+        {
+
```
Comment (afrozchakure) [inline Codewars/Leap_years.cpp:18]: Thanks. Just make sure that your code is formatted correctly, it is considered good practice and makes it easy for others to read your code.
Auto Format option will be available in VS Code and Sublime by default or you could install an extension for .cpp.

**Review by afrozchakure — CHANGES_REQUESTED** [review]
Thanks for contributing again @soham4abc :tada: .

Remove the extra spaces. It helps others who later refer your code. :smile:

**Comment by afrozchakure** [comment]
@soham4abc do let me know if you are working on it ?

**Comment by soham4abc** [comment]
Yes I am working on it, I am a bit slow cause of my college assignments ... Sorry for that... Will do it soon.... Plus what kind of formatting do you want?...

**Comment by soham4abc** [comment]
I have done the changes as per request. Please do see it and let me know it requires more changes or not!

**Review by afrozchakure — APPROVED** [review]

**Comment by afrozchakure** [comment]
LGTM :tada: :+1:

**Pull request merged**